Author: Tzafrir Cohen <tzafrir.cohen@xorcom.com>
Description: : Mark the two extra ARM ABIs as linux-gnu

Will be included in asterisk 1.8.6 .

--- a/main/Makefile
+++ b/main/Makefile
@@ -34,7 +34,7 @@ AST_LIBS += $(OPENSSL_LIB)
 AST_LIBS += $(BKTR_LIB)
 AST_LIBS += $(LIBXML2_LIB) 
 
-ifneq ($(findstring $(OSARCH), linux-gnu uclinux linux-uclibc linux-gnueabi kfreebsd-gnu),)
+ifneq ($(findstring $(OSARCH), linux-gnu linux-gnueabi linux-gnueabihf clinux linux-uclibc kfreebsd-gnu),)
   ifneq ($(findstring LOADABLE_MODULES,$(MENUSELECT_CFLAGS)),)
   AST_LIBS+=-ldl
   endif
